Keas.Mvc/ClientApp/components/Access/AccessDetails.tsx

```diff
@@ -9,22 +9,24 @@ import {
     ListGroup,
     ListGroupItem
 } from "reactstrap";
-import { IAccess } from "ClientApp/Types";
+import { IAccess, IAccessAssignment } from "ClientApp/Types";
 import AccessEditValues from "./AccessEditValues";
 
 
 interface IProps {
     modal: boolean;
     closeModal: () => void;
     selectedAccess: IAccess;
+    onRevoke: (accessAssignment: IAccessAssignment) => void;
 }
 
 
 export default class AccessDetails extends React.Component<IProps, {}> {
 
     public render() {
-        if (this.props.selectedAccess == null)
+        if (this.props.selectedAccess == null){
```

Comment:
@jcubbage Try to avoid the double equals -- i think you could either go with `this.props.selectedAccess === null` or simply `!this.props.selectedAccess`, depending on if you really need to check for null or just a falsey value.  I always love the later but it is semantically ever so slightly different

Reply:
So, this was actually TSLint complaining about the missing curly brackets from someone else. I'll update to triple equal.